File: PYTHON Exercises1/exercise4.py
import math
a = input ('Enter value for a: ')
b = input ('Enter value for b: ')
c = input ('Enter value for c: ')
a = int(a)
b = int(b)
c = int(c)
if (b**2 - 4*a*c)<0 :
    print ('No real-valued solutions exist')
else :
    x = (math. sqrt (b**2 - 4*a*c) - b) / (2*a)
    y=  (-(math. sqrt (b**2 - 4*a*c)) - b) / (2*a)
    print ('The solutions of the quadratic equation are: ' , x , 'and', y)

File: 1.Intoduction to Java/From Python to Java/Exercise4.java
import java.util.Scanner;
class Exercise4 {
	public static void main (String args[]){
		Scanner scanner = new Scanner(System.in);
		System.out.print ("Enter value for a:");
		int a = scanner.nextInt();
		System.out.print ("Enter value for b:");
		int b = scanner.nextInt();
		System.out.print ("Enter value for c:");
		int c = scanner.nextInt();

		if (Math.pow(b,2) - 4*a*c <0) {
			System.out.println ("No real-valued solutions exist.");
		}else {
			double x = (Math.sqrt(Math.pow(b,2) - 4*a*c) - b) / (2*a);
            double y = (-(Math. sqrt (Math.pow(b,2) - 4*a*c)) - b) / (2*a);
            System.out.println ("The solutions of the quadratic equation are: " + x + " and " + y);
		}


	}
}